templates/core/features/index.html

```diff
@@ -202,9 +151,168 @@ <h2>Recovery mode</h2>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Enable resellers to rebrand devices</li>
+        <li class="p-list__item is-ticked">Easy migration path between UC20 and UC22</li>
+      </ul>
+      <p>
+        <a href="">Learn more about remodelling&nbsp;&rsaquo;</a>
+      </p>
+    </div>
+    <div class="col-4 u-align--center u-vertically-center u-hide--medium u-hide--small">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/25c42877-simplified-software-management-2.svg",
+        alt="",
+        width="150",
+        height="150",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
 </section>
 
 <section class="p-strip">
+  <div class="u-fixed-width">
+    <h2>What's under the hood</h2>
+    <p class="u-sv3">Ubuntu Core is ideal for embedded devices because it manages itself.  Snaps, Snapd and Snapcraft bring security and robustness. Applications are easy to install, easy to maintain, and easy to upgrade.</p>
+  </div>
+  
+  <div class="row">
+    <div class="col-12">
+      <div class="p-tabs">
+        <div class="p-tabs__list js-tabbed-content" role="tablist" aria-label="What's under the hood">
+          <div class="p-tabs__item">
+            <button class="p-tabs__link" role="tab" aria-selected="true" aria-controls="snaps-tab" id="snaps-tab-cta">Snaps</button>
+          </div>
+          <div class="p-tabs__item">
+            <button class="p-tabs__link" role="tab" aria-selected="false" aria-controls="snapd-tab" id="snapd-tab-cta" tabindex="-1">Snapd</button>
+          </div>
+          <div class="p-tabs__item">
+            <button class="p-tabs__link" role="tab" aria-selected="false" aria-controls="snapcraft-tab" id="snapcraft-tab-cta" tabindex="-1">Snapcraft</button>
+          </div>
+        </div>
+
+        <div tabindex="0" role="tabpanel" id="snaps-tab" aria-labelledby="snaps-tab-cta">
+          <div class="row u-equal-height">
+            <div class="col-8">
+              <h3>Snaps</h3>
+              <p>Ubuntu Core is built from snaps, a secure, confined, dependency-free, cross-platform Linux packaging format. Snaps are entirely self-contained, even to the point of encapsulating their own file system. This means they include everything they need to run in any environment. They're used by Ubuntu Core to both compose the image that's run on a device, and to deliver consistent and reliable software updates, even to low-powered, inaccessible and remotely administered embedded and IoT systems.</p>
+              <p>
+                <a href="/core/docs/snaps-in-ubuntu-core">Learn more about snaps&nbsp;&rsaquo;</a>
+              </p>
+            </div>
+            <div class="col-4 u-vertically-center u-hide--medium u-hide--small u-align--center">
+              {{ image (
+                url="https://assets.ubuntu.com/v1/a41bcb2f-UC20_Snapcraft_2.svg",
```

Comment:
The image doesn't match with the image on copy doc.
```suggestion
                url="https://assets.ubuntu.com/v1/7de21966-snapshot.svg",
```

Reply:
Ah yes - updated!